Gaijinizer patch/CommonEvents_122.txt

```diff
@@ -4,85 +4,91 @@
 
 何を着けてるのかなー？
 > CONTEXT: CommonEvents_122/5/Dialogue/enemy2
-Oh? \\N[1]-kun. What are you wearing?#MTLed
+Oh? \\N[1]-kun. What's that you're wearing?#Ron
 > END STRING
 
 > BEGIN STRING
 もしかして、そういうことなの？
 
 私に遊んで欲しいんだぁ…？♡
 > CONTEXT: CommonEvents_122/9/Dialogue/enemy2
-Is that what you mean? Do you want me to play
-...? Moth#MTLed
+Is that what I think it is? Do you want me to play
+with you...? ♡#Ron
 > END STRING
 
 > BEGIN STRING
 な……何を言ってる………！！
 > CONTEXT: CommonEvents_122/13/Dialogue/kazuya
-N ...... What are you saying.........! !#MTLed
+Wh-... What are you talking about...!!#Ron
 > END STRING
 
 > BEGIN STRING
 あれぇ？それとも気付いてないんだ？
 
 それじゃあ、分からせてあげる…。
 > CONTEXT: CommonEvents_122/15/Dialogue/enemy2
-? Or do you not notice? Well then let me know
-....#MTLed
+Hmm? Haven't you realised?
+
+Well then let me fill you in...#Ron
 > END STRING
 
 > BEGIN STRING
 四つん這いになりなさい♪
 > CONTEXT: CommonEvents_122/20/Dialogue/enemy2
-Become a quarrel♪#MTLed
+Get on all fours. ♪#Ron
 > END STRING
 
 > BEGIN STRING
 えっ……！？
 
 な、なに……！！
 > CONTEXT: CommonEvents_122/24/Dialogue/kazuya-0
-eh……! ? ? ...! !#MTLed
+Huh...!??
+
+Wh- what is this?!#Ron
 > END STRING
 
 > BEGIN STRING
 クスクス…♡
 
 それじゃあ次は、転げ回りながら服を脱ごっか♡
 > CONTEXT: CommonEvents_122/28/Dialogue/enemy2
-* Giggle *...♡. Well then, I'll take off my
-clothes while rolling around#MTLed
+*Giggle*...♡
+Okay, next roll over while taking your clothes
+off.#Ron
 > END STRING
 
 > BEGIN STRING
 えっ…えっ……！！
 
 な、なにこれぇ……！！
 > CONTEXT: CommonEvents_122/33/Dialogue/kazuya-0
-Eh...Eh ...!!What's this ......? !#MTLed
+Huh... What!!
+
+W- what's happening?!#Ron
 > END STRING
 
 > BEGIN STRING
 うるさいなぁ……せっかく犬っぽくなってきたんだから
 
 今後の言葉はぜんぶワンって言いなさい。
 > CONTEXT: CommonEvents_122/37/Dialogue/enemy3
-Noisy...Because I'm getting dog-like
-Please say all words from now on are one.#MTLed
+Quiet down... You're supposed to be like a dog
+so from now on you're only allowed to bark.#Ron
```

Comment:
It'd probably be best to split up the second sentence into two:
>_**Quiet down... you're supposed to act like a dog.
So from now on, you're only allowed to bark.**_

(We could probably interpret "Quiet down" as "Shush" or "Shut up", but honestly that depends on the tone you wish to take.)